vclib/src/commonMain/kotlin/at/asitplus/wallet/lib/iso/cborSerializer.kt

```diff
@@ -1,7 +1,46 @@
 package at.asitplus.wallet.lib.iso
 
+import at.asitplus.wallet.lib.SerializerLookup
+import at.asitplus.wallet.lib.ItemValueDecoder
+import at.asitplus.wallet.lib.ItemValueEncoder
 import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.KSerializer
 import kotlinx.serialization.cbor.Cbor
+import kotlinx.serialization.descriptors.SerialDescriptor
+import kotlinx.serialization.encoding.CompositeDecoder
+import kotlinx.serialization.encoding.CompositeEncoder
+
+object Cbor {
```

Comment:
Again, maybe use a more expressive name, as we have `Cbor` also in KMP-Crypto, for example